app/models/company.rb

```diff
@@ -0,0 +1,6 @@
+class Company < ActiveRecord::Base
+  validates :name, :organization, :title, :state, :city, presence: true
+  validates :email, presence: true, length: { maximum: 255 },
+                    format: { with: /\A[\w+\-.]+@[a-z\d\-.]+\.[a-z]+\z/i, }
+
+end
```

Comment:
Nice and clean validations! I'm interested to see how the layout comes back but do you think we should add in the number of hires they are committing to as well?